examples/compute-dg-api.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    absolute_df
</pre><p>This is great! It's an often requested operation, so it's good to be able to show it off here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/cinnabar/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>